### ISSUE TITLE
Enhance `:focus-visible` states color contrast

### ISSUE THREAD

**Issue by ffoodd** [Orange-OpenSource/Orange-Boosted-Bootstrap#398]
Our primary color `#f16e00` leads to insufficient contrasts against `#ffffff`.

Thanks to the `:focus-visible` polyfill, we're now able to distinguish the focus state when using keyboard—which already got enhanced with `outline`s.

I suggest we go a step further by ensuring contrasts (mostly using a darker orange) for components using `#f16e00` in either `color` or `background-color` when focused.

- [ ] inventory cases with `#f16e00` when focused
- [ ] select the minimum viable orange: for now, `$orange-700` which results in `#a44b00`, see [current `v5-dev` deploy](https://v5-dev--boosted.netlify.app/docs/5.0/customize/color/#all-colors)
- [ ] apply it in listed cases.
